index.js

```diff
@@ -54,6 +55,11 @@ function handleArgs(cmd, args, opts) {
 		opts.env = npmRunPath.env(Object.assign({}, opts, {cwd: opts.localDir}));
 	}
 
+	if (process.platform === 'win32' && (parsed.command === process.env.comspec || path.basename(parsed.command) === 'cmd.exe')) {
```

Comment:
Is this check ok? Do we need the `parsed.command === process.env.comspec` check or is the basename always `cmd.exe`?

Reply:
> is the basename always cmd.exe?

Basename should always be `cmd.exe`. I think only the location could change, hence `comspec`.